opencti-platform/opencti-front/src/private/components/data/connectors/ConnectorWorksErrorLine.tsx

```diff
@@ -70,12 +90,60 @@ const ConnectorWorksErrorLine: FunctionComponent<ConnectorWorksErrorLineProps> =
         TransitionComponent={Transition}
         onClose={handleToggleModalError}
       >
-        <DialogTitle>{t_i18n('Error')}</DialogTitle>
-        <DialogContent>
+        <DialogTitle>{t_i18n('Details')}</DialogTitle>
+        <DialogContent sx={{ minWidth: '500px' }}>
           <DialogContentText>
-            <pre><ItemCopy content={error.rawError.timestamp ?? '-'} /></pre>
-            <pre><ItemCopy content={error.rawError.message ?? '-'} variant={'wrap'} /></pre>
-            <pre><ItemCopy content={error.rawError.source ?? '-'} variant={'wrap'} /></pre>
+            <Typography variant="h4" gutterBottom={true}>
+              {t_i18n('Entities')}
```

Comment:
shall this be "Source" instead of "Entities"?

Reply:
Done